spec/models/game_spec.rb

```diff
@@ -20,4 +20,135 @@
       expect(game.pieces.last.color).to eq 'white'
     end
   end
+
+  describe 'check?' do
+    let(:empty_game) { FactoryGirl.create(:game) }
+    let(:blk_king) do
+      empty_game.pieces.create(
+        type: 'King',
+        color: 'black',
+        x_position: 3,
+        y_position: 4,
+        moved: true
+      )
+    end
+    let(:wht_king) do
+      empty_game.pieces.create(
+        type: 'King',
+        color: 'white',
+        x_position: 6,
+        y_position: 6,
+        moved: true
+      )
+    end
+    let(:queen) do
+      empty_game.pieces.create(
+        type: 'Queen',
+        color: 'white',
+        x_position: 1,
+        y_position: 5,
+        moved: true
+      )
+    end
+    let(:knight) do
+      empty_game.pieces.create(
+        type: 'Knight',
+        color: 'white',
+        x_position: 6,
+        y_position: 7,
+        moved: false
+      )
+    end
+
+    it 'should return true if King is under check horizontally' do
+      expect(queen.move!(1, 4)).to eq true
```

Comment:
You should try to limit to one `expect` per `it`. Otherwise, one condition failing means that the rest of the suite will not run and you will not have a clear picture of what is actually broken.